samples/WebApi/README.md

```diff
@@ -57,34 +57,7 @@ services.AddMvcCore().AddJsonFormatters();
 
 Replacing `AddMvc()` with `AddMvcCore()` adds only the basic MVC functionality. It's followed by explicit registration of services, which are required by the application - in this case the `JsonFormatter`. For more details see [Fabian Gosebrink's blog post comparing AddMvc and AddMvcCore](https://dzone.com/articles/the-difference-between-addmvc-and-addmvccore).
 
-## Using reflection 
```

Comment:
Given that you are completely removing all references to rd.xml from this sample, it might nice to add some kind of 'Potential issues with Reflection' section that refers, for example, either to a previous commit or to the MonoGame sample. Just to let people know what to do if they add new code/components that cannot be handled well by the improved IL scanner.

Reply:
Added two links for what to do with reflection